CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+---
+CIP: ???
+Title: On-Chain DRep Credential & Extended Governance NFTs
+Status: Draft
+Category: Tokens
+Authors:
+  - "Derrick Oatway <alpine@malamaproject.org>"
+Implementors:
+  - DRep Collective
+Discussions:
+  - "https://github.com/cardano-foundation/CIPs/pull/1007"
+Created: "2025-03-01"
+License: CC-BY-4.0
+---
+
+## Abstract
+
+> "**Poets are the unacknowledged legislators of the world.**"  
+> — Percy Bysshe Shelley, _A Defence of Poetry_ (1821)
+
```

Comment:
```suggestion
```
I recalled that the epigraphs were being removed as part of our usual prohibition of subjective language.  @Alpine-Oracle as originally told @Crypto2099 in our behind-the-scenes initial review I think these are apt choices for quotes... and I do think they should be included in some other writing about this CIP.